### PR TITLE
Update meta.yaml

### PR DESCRIPTION
**Pull request by rpanai**
bump version

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,47 +1,44 @@
...
-    - submitit.core
-    - submitit.local
-    - submitit.slurm
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
-  home: https://github.com/facebookincubator/submitit
```
Comment (rpanai) [inline recipe/meta.yaml:37]: add this

**Comment by Ubadub** [comment]
Can you recreate this as a branch of a fork? @rpanai

File: recipe/meta.yaml (hunk 1 above)
Comment (Ubadub) [reply recipe/meta.yaml:37]: Why delete the other imports? Just wondering

**Comment by Ubadub** [comment]
@rpanai dude, did you merge this? Or push to main directly? You're not supposed to do that, can you revert the commit? Also, it won't get updated until you re-render

File: recipe/meta.yaml (hunk 1 above)
Comment (rpanai) [reply recipe/meta.yaml:37]: As many things changed in this library, the reason why it didn't upgrade automatically, I rebuild the recipe. I guess I did a mess.

**Comment by rpanai** [comment]
@conda-forge-admin, please rerender

**Comment by Ubadub** [comment]
@rpanai You need to fork this, and open the PR from a branch on the fork. The bot is not going to comply otherwise.

**Comment by Ubadub** [comment]
Would you like me to handle it? I can.

**Pull request closed without merging**